### DISCUSSION TITLE
OBC Sector flag color overlay

### DISCUSSION THREAD

**Discussion by JJWatMyself** [f1multiviewer/issue-tracker#169] [Feature requests]
The idea is that for OBC video streams, when driver is in a sector with a yellow flag, that there would be a yellow flag overlay on the OBC to show the driver was in a yellow flag sector. And then when they leave that sector, or if when in the sector it goes green, that the flag overlay show a green green. 

![image](https://user-images.githubusercontent.com/22590445/187102248-999b7e19-2ffb-4a11-ba1b-f864a94f6cf8.png)

![image](https://user-images.githubusercontent.com/22590445/187102265-1ef29f3f-40f0-4bd7-b8a9-0911283ad2c8.png)

Possibly the flag would be the the same as the digiflag panel, which is what the driver sees, track side / on their steering wheel.

Could also do the same thing when a driver is blue flagged.

**Comment by LapsTimeOFF** [comment]
Blue flag will be the most simple to add because the driver number is in the RCM data, but the yellow green... is more difficult
